lib/scripts/common/mesosflexinstall

```diff
@@ -51,10 +53,12 @@ function install_mesos {
 function configure_slave {
   if [[ ${slave_hostname+isset} ]]
   then
-    mkdir -p /etc/mesos-slave
+    mkdir -p /etc/mesos-slave/resources
     echo "$slave_hostname" > /etc/mesos-slave/hostname
     echo "docker,mesos" > /etc/mesos-slave/containerizers
     echo "5mins" > /etc/mesos-slave/executor_registration_timeout
+    echo "$slave_cpus" > /etc/mesos-slave/resources/cpus
+    echo "$slave_mem" > /etc/mesos-slave/resources/mem
```

Comment:
I was having a problem where the slave cpu was 1 and mem is 1GB, no matter how much of those I set on the master.